README.md

```diff
@@ -4,10 +4,12 @@ Reverse proxy app to combine multiple applications and redirect rules to create
 
 ## Configuration
 
-Set the `PROXIES` environment variable to a JSON object mapping proxy routes to URLs:
+Set the `PROXIES` environment variable to a JSON object mapping proxy routes to URLs. As an example, you can do this manually with a command similar to this:
 
     cf set-env fec-proxy PROXIES='{"/": "https://app1.18f.gov", "/app2": "https://app1.18f.gov"}'
```

Comment:
Should update the proxies to be this syntax as we are using .app.cloud.gov

`cf set-env fec-proxy PROXIES='{"/": "https://app1.app.cloud.gov", "/app2": "https://app1.app.cloud.gov"}'`